src/repository/fetchJson.js

```diff
@@ -0,0 +1,13 @@
+import fetch from 'node-fetch';
+
+const fetchJson = url => fetch(url)
+  .then((res) => {
+    if (res.status === 200) {
+      return res;
+    }
+
+    throw new Error(res.statusText);
```

Comment:
`Boom` przy tym rozmiarze projektu wydaje mi się być overkillem.
Może faktycznie lepiej zostawić jak jest, a gdyby potrzebny był bardziej poważny error handling to pomyśleć nad stworzeniem typu errora i dopisaniem mu paru atrybutów z kodem itd